homeassistant/components/influxdb.py

```diff
@@ -154,8 +154,9 @@ def event_to_json(event):
             return
 
         try:
-            if (whitelist_e and state.entity_id not in whitelist_e) or \
-                    (whitelist_d and state.domain not in whitelist_d):
+            if ((whitelist_e or whitelist_d)
```

Comment:
Why is influxdb using their own implementation instead of our `helpers.entityfilter` ?

Reply:
It appears the helper was implemented without completing the conversion of all components.

https://github.com/home-assistant/home-assistant/pull/10221#issuecomment-340656617